daml-lf/encoder/src/test/scala/com/digitalasset/daml/lf/archive/testing/DamlLfEncoderTest.scala

```diff
@@ -26,29 +26,31 @@ class DamlLfEncoderTest
     "be readable" in {
 
       val modules_1_0 = Set[DottedName](
-        "Bool",
-        "Date",
-        "Decimal",
-        "Int64",
-        "List",
-        "Party",
-        "Record",
-        "Text",
-        "Timestamp",
-        "Unit",
-        "Variant")
-
-      val modules_1_1 = modules_1_0 + "Option"
-      val modules_1_3 = modules_1_1 + "TextMap"
-      val module_1_7 = modules_1_3 + "Enum"
-      val modules_1_dev = module_1_7
+        "UnitMod",
```

Comment:
An alternative to this enumeration may be to `map` over the original module names and add the postfix (if it is a uniform one).